serverless/serverless
# Spotinst: Added support for CORS for Functions

## What did you implement:
Added support for CORS in functions + documentation.
Updated Spotinst templates.



## How did you implement it:
Updated serverless.yml file for python, ruby, node, and java.
Added docs.


## How can we verify it:
See: /serverless/docs/providers/spotinst/guide/cors.md
        /serverless/lib/plugins/create/templates


## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
